openapi/openapi.yml

```
@@ -927,7 +927,7 @@ paths:
   '/project/{project_id}/location/{location}/postgres':
     get:
       operationId: listLocationPostgresDatabases
-      summary: List Postgres Databases in a specific location of a project
+      summary: List postgres databases in a location
```

I recommend using `PostgreSQL` instead of `postgres` here and in other descriptions.

```suggestion
      summary: List PostgreSQL databases in a location
```

---

I replaced them